src/Service/Sqs/src/Enum/QueueAttributeName.php

```diff
@@ -2,7 +2,7 @@
 
 namespace AsyncAws\Sqs\Enum;
 
-class QueueAttributeName
+final class QueueAttributeName
```

Comment:
This is a bug fix. All our enums and value objects are final

Reply:
I don't know why generator update this file now and didn't triggered error before